File: year2020/month10/day1019/ms_1019.cs
using System;

namespace algorithm_81
{
    class ms_1019
    {
        static void Main(string[] args)
        {
            int n;

            while(true) 
            {
                Console.Write("ASCII code =? ");
                n = Convert.ToInt32(Console.ReadLine());

                if(33 <= n && n <= 127)
                    Console.WriteLine(Convert.ToChar(n));
                else break;
            }
        }
    }
}

File: year2020/month10/day1019/ms_1019.cpp
#include <iostream>

using namespace std;

int main() {
	int n;

	while (true) {
		cout << "ASCII code =? ";
		cin >> n;

		if (33 <= n && n <= 127) {
			cout << (char)n << endl;
		}
		else break;
	}

	return 0;
}